src/autoscaler/eventgenerator/aggregator/policyPoller_test.go

```diff
@@ -79,13 +79,13 @@ var _ = Describe("PolicyPoller", func() {
 					clock.Increment(1 * testPolicyPollerInterval)
 					Eventually(database.RetrievePoliciesCallCount).Should(BeNumerically(">=", 2))
 					policyMap := poller.GetPolicies()
-					Expect(policyMap[testAppId1]).To(Equal(&models.AppPolicy{
-						AppId: testAppId1,
+					Expect(policyMap[testAppId]).To(Equal(&models.AppPolicy{
+						AppId: testAppId,
 						ScalingPolicy: &models.ScalingPolicy{
 							InstanceMax: 5,
 							InstanceMin: 1,
 							ScalingRules: []*models.ScalingRule{{
-								MetricType:            models.MetricNameMemory,
+								MetricType:            "test-metric-name",
```

Comment:
modes.??? constant is not used to test generic metric name.  There is only one place using string "test-metric-name",  it is not worth defining it as a constant or variable.